SubmissionExample.ipynb

```diff
@@ -0,0 +1,592 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## End to End Mlperf Submission example\n",
+    "\n",
+    "This is following the [General MLPerf Submission Rules](https://github.com/mlperf/policies/blob/master/submission_rules.adoc).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get the mlperf source code\n",
+    "\n",
+    "You run this notebook from the root of the mlperf inference tree that you cloned with\n",
+    "```\n",
+    "git clone https://github.com/mlperf/inference.git --depth 1\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Build loadgen"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# build loadgen\n",
+    "!pip install pybind11\n",
+    "!cd loadgen; CFLAGS=\"-std=c++14 -O3\" python setup.py develop"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!cd v0.5/classification_and_detection; python setup.py develop"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Working Directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/mnt/sdb/home/ipython/notebooks/inference/v0.5/classification_and_detection\n"
+     ]
+    }
+   ],
+   "source": [
+    "%cd v0.5/classification_and_detection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "## download imagenet and coco.\n",
+    "## we keep a copy under /data so just symlink it.\n",
+    "mkdir data\n",
+    "ln -s  /data/imagenet2012 data/\n",
+    "ln -s  /data/coco data/"
```

Comment:
this may be worth calling out or mentioning as something you probably need to change